### PR TITLE
Popup window DOM

### PR DESCRIPTION
**Pull request by AbdifatahYasin1**
### Pop-up window
This project I used JavaScript Dom to make Pop-up window.

### what is Pop-up
A pop-up is a graphical user interface ( GUI ) display area, usually a small window, that suddenly appears ("pops up") in the foreground of the visual interface.

Coding 👩‍💻

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,10 +21,15 @@ in microverse bootcamp. It is html and CSS with no frameworks.
...
-clone the project
-Run on your browser
+You can have a copy of this repo run on your local machine by just directly cloning it
+
+> Cloning steps:
+
+1. - [ ] Open the `Terminal`
+2. - [ ] Navigate to the directory where you want to add this repository
+3. - [ ] Run `https://github.com/AbdifatahYasin1/My-portfolio-Web.git`
+4. - [ ] Run `npm install` to install the necessary modules
```
Comment (christianbiring1) [inline README.md]: - [ ] Kindly make sure you are specific and clear on the instructions to start with your project locally.
Hint: You may say
- open up the terminal
- Cloning with `git clone https://github.com/AbdifatahYasin1/My-portfolio-Web.git`
- cd My-portfolio-Web
- Open in the IDE
- open with the liver server

File: README.md [hunk 2]
```diff
@@ -34,7 +39,10 @@ To get a local copy up and running follow these simple example steps.
 - Twitter: [@CabdifataaxYuusuf](https://twitter.com/CabdifataaxYy)
 - LinkedIn: [Cabdifataax Yaasiin](https://www.linkedin.com/in/cabdifataax-yaasiin-69977019a/)
 
-
+👤 **Tonny Tei**
+-Github: [@teitonny](https://github.com/teitonny)
+-Twiter: [@TonnyTei](https://twitter.com/TonnyTei)
+-linkedIn: [TonnyTei](https://www.linkedin.com/in/tonny-tei-38a4771a0/)
```
Comment (christianbiring1) [inline README.md]: Are these your social media handle?
- [ ] Kindly make sure you put the correct link to your social media.

File: style.css [hunk 3]
```diff
@@ -455,102 +457,329 @@ input {
...
+  }
+
+  .Popup-section .Popup-container {
+    margin: 2% 5% 2% 4%;
+    background: #fff;
+    border-radius: 20px;
+    position: relative;
+    max-height: 100%;
+    display: none;
+  }
```
Comment (christianbiring1) [inline style.css]: - [ ] KIndly make sure your pop-up is well-centered following the width and height of the page as is the case on the Figma template.
- [ ] Also make sure you use the same color as the Figma design for the `blur`. 
Ypur design :point_down: 
image

Figma design :point_down: 
image

File: style.css [hunk 4]
```diff
@@ -455,102 +457,329 @@ input {
...
+  .Popup-container h2 {
+    text-align: center;
+    font-size: 20px;
+  }
+
+  .PopUp-item {
     display: flex;
+    gap: 20px;
+    font-size: 14px;
+    justify-content: center;
```
Comment (christianbiring1) [inline style.css]: - [ ] Kindly make sure you use the correct spacing, color, and typo (font, margin, padding) as the provided template in Figma.
Your design :point_down: 
image

Figma template :point_down: 

image

**Review by christianbiring1 — CHANGES_REQUESTED** [review]
Hi @AbdifatahYasin1 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
# STATUS: REQUEST CHANGES :recycle: 

163947610-e4d849d5-e69c-49b9-90bd-a4191f4e2856

## Highlights
- [ ] All linters checked are green :heavy_check_mark: 
- [ ] Descriptive PR `title` and `summary` :+1: 
- [ ] Descriptive commit message.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Make your page responsive on mobile. as you can see your designn :point_down: 
image

Compared to the provided template in Figma :point_down: 
image


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 3 above)
Comment (AbdifatahYasin1) [reply style.css]: @christianbiring1  I did not get that how ?

**Review by sumairq — APPROVED** [review]
### Hi @AbdifatahYasin1 , 



Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✔️ Good commit history
✔️Linters are passing
✔️Very professional README
✔️Correct Github flow
✔️Descriptive PR
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**